### DISCUSSION TITLE
433 tx

### DISCUSSION THREAD

**Discussion by electro991** [CrashOverride85/zc95#103] [Q&A] [answered]
I am looking for someone that fitted a working 433 tx module to J11 for a picture of the module used. i would like to source one but there are a few types available. thank you

**Comment by CrashOverride85** [comment, marked as answer]
The one I've used looks the same as the transmitter in this pair:
https://www.ebay.co.uk/itm/283886821529

But any of the similar 433mhz AM OOK transmitter modules with the same pinout should work

**Reply by electro991** [reply]
Thank you.

**Comment by electro991** [comment]
For the record, I have received these and they did not work. box doen't start when connected. Not sure why.
![image](https://github.com/CrashOverride85/zc95/assets/173577055/e17fd648-ea04-47bb-b091-fdffb39d204a)

**Reply by CrashOverride85** [reply]
Interesting, that looks very similar to what I've got, which works fine:
![DSC_3616](https://github.com/CrashOverride85/zc95/assets/56609106/bf9d7dcd-325f-47f7-b47a-984f3bd88893)

**Reply by electro991** [reply]
Looks like it is now working ! Good thing I ordered 5, 2nd one worked ! They are identical.
